.github/actions/run-backend-tests/action.yml

```diff
@@ -154,22 +155,10 @@ runs:
           run: | # async_migrations covered in ci-async-migrations.yml
               pytest ${{
                   inputs.person-on-events == 'true'
-                  && './posthog/clickhouse/ ./posthog/hogql/ ./posthog/queries/ ./posthog/api/test/test_insight* ./posthog/api/test/dashboards/test_dashboard.py'
```

Comment:
Why is `./posthog/hogql/` exempt in the person-on-events case?

Reply:
https://posthog.slack.com/archives/C0113360FFV/p1713436666421219?thread_ts=1713349567.878239&cid=C0113360FFV

Reply:
To save time. Tests in the `hogql` and `hogql_queries` folders flip the PoE flags on and off as needed, so there's no point to run them twice.